README.md

```diff
@@ -39,7 +39,8 @@ flowchart TB
     2402-->3000
     2402-->3005
     2402-->3007
-    2402-->3804
```

Comment:
Can we add this line back?

![image](https://user-images.githubusercontent.com/14791619/177388430-8530c3d4-251d-4df3-83d7-b298f6176e56.png)

Reply:
oof sorry I thought that said 3004 lol